atlas/AtlasMl/atlasml/ml/clustering.py

```
@@ -0,0 +1,70 @@
+    euclidean = "euclidean"
+    cosine = "cosine"
+    jaccard = "jaccard"
+
+
+def apply_hdbscan(
+    matrix: np.ndarray, 
+    eps: float = 0.5, 
+    min_samples: int = 5, 
+    metric: str = "euclidean", 
+    min_cluster_size: int = 10
+) -> Tuple[np.ndarray, Optional[np.ndarray], Optional[np.ndarray]]:
```

Ruff (0.12.2)

Use `tuple` instead of `Tuple` for type annotation

Replace with `tuple`

(UP006)